### ISSUE TITLE
satdump without ui need help

### ISSUE THREAD

**Issue by dl5raz** [SatDump/SatDump#780]
I have problem to find out parameterstring : satdump ...?.. for Meteor wxsat
I use sdrplay on linumint, satdump-ui workes ok 
Want  using  satdump on a raspi4  without  ui to  reduce  workload  on  the raspi4
Example:
/usr/bin/satdump live noaa_apt .  --source "RSP1A 190309B796" --samplerate 1.024e6 --frequency 137.1000e6 --satellite_number 19 --sdrpp_noise_reduction --source_id 00000001 --gain 29.7 
**Error: (E) Could not find a handler for source type : RSP1A 190309B796!**
But in GUI RSP1A 190309B796  is working!
what is wrong?

**Comment by JVital2013** [comment]
Hello! Your command should be (roughly):

```
satdump live noaa_apt . --source sdrplay --samplerate 2e6 --frequency 137.1000e6 --satellite_number 19 --sdrpp_noise_reduction --source_id 190309B796
```
If that does not work, try it without the source_id parameter (in case there's something wrong with the ID; `satdump sdr_probe` will tell you what it should be.

Make sure to verify a valid samplerate in the UI as well. If I recall correctly, 1.024e6 is not one of the samplerates the sdrplay can handle (so I changed it to 2 in my example command)

You will also need to specify `--lna_gain` and `--if_gain`. SDRPlay SDRs don't have a general gain like it is often presented to the user for RTL-SDR, so you need to specify both of the individual gain controls, based on what works in the UI

**Comment by dl5raz** [comment]
thanks a lot! have now a working command line:
/usr/bin/satdump live noaa_apt /home/mint/Bilder  --source sdrplay --samplerate 2e6 --frequency 137.100e6 --satellite_number 19 --sdrpp_noise_reduction  --lna_gain 9 --if_gain 54  --start_timestamp 1723576491 --finish_processing --timeout 978
next question: is there a way to use the internal satdump scheduler to make things automatic? 
example would be nice for the two meteor sats!

**Comment by JVital2013** [comment]
Yes, autotrack can be used via the CLI. For now, you need to manually compose a .json file to tell it what to track, but soon a feature will be added to export CLI autotrack configurations from the UI.

You can read up on configuring the json manually at https://www.a-centauri.com/articoli/automating-satellite-reception-with-satdump-cli.

**Comment by JVital2013** [comment]
The ability to export cli autotrack configs from the UI is now available in the latest nightlies, and the soon-to-be released SatDump 1.2.1

![image](https://github.com/user-attachments/assets/26898dbd-aa3f-42b5-b376-35dd953184a3)
